### PR TITLE
Cu 9auf6m create location page

### PR DESCRIPTION
**Pull request by Hamza-A**
## Description
This PR aims to implement the `/account/locale` page in which the user could set the language of the application

- I created `<RadioButton />` components and its stories
- I created `<MagnifyingGlass />` icon and its story
- I created `<Locale />` page component in which I implemented those previous components
- I updated the routes in the nav bars
- I written tests
## Type of change

- [ ] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [x] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## How Has This Been Tested?
I created a new test file to test the whole `<LocalePage />` component

- I'm checking if there is an input
- I'm checking if there is a single list item "English" with a checked `<RadioButton />`
## Screenshots
image
image

## Checklist:

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

File: src/components/display/fewlines/Languages/Locale.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          </div>
+        </div>
+      </Container>
+      <LanguagesList>
+        <ListItem>
+          <Value>English</Value>
+          <RadioButton name="locale" checked={true} />
+        </ListItem>
+        <Separator />
+      </LanguagesList>
```
Comment (Fuzznimp) [inline src/components/display/fewlines/Languages/Locale.tsx:29]: ❓ Why did you choose to put the language list outside the container?

File: src/components/display/fewlines/RadioButton/RadioButton.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  );
+};
+
+export const UncheckedRadioButton = (): JSX.Element => {
+  return (
+    <>
+      <RadioButton name="story" checked={false} />
+    </>
+  );
+};
```
Comment (Fuzznimp) [inline src/components/display/fewlines/RadioButton/RadioButton.stories.tsx:21]: You don't need React Fragments here.

File: src/components/display/fewlines/RadioButton/RadioButton.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+};
+
+const Wrapper = styled.div`
+  .radio-container {
+    position: relative;
+    cursor: pointer;
+    user-select: none;
+    padding-left: 3rem;
+  }
+
```
Comment (Fuzznimp) [inline src/components/display/fewlines/RadioButton/RadioButton.tsx:27]: You should remove the `className` and target the html tag right away.
```suggestion
export const RadioButton: React.FC<RadioButtonProps> = ({ name, checked }) => {
  return (
    <Wrapper>
      <label>
        <input type="radio" name={name} defaultChecked={checked} />
        <span/>
      </label>
    </Wrapper>
  );
};

const Wrapper = styled.div`
	label {
		...
	}
	
	span {
		...
	}

```

**Review by Fuzznimp — CHANGES_REQUESTED** [review]

File: src/components/display/fewlines/Languages/Locale.tsx (hunk 1 above)
Comment (Hamza-A) [reply src/components/display/fewlines/Languages/Locale.tsx:29]: On our wireframe, the `<H1 />` and the `<Input />` are contained in a wrapper with a maximum `width` whereas the list stretches to the entire `width` of the page

image

File: src/components/display/fewlines/RadioButton/RadioButton.stories.tsx (hunk 2 above)
Comment (Hamza-A) [reply src/components/display/fewlines/RadioButton/RadioButton.stories.tsx]: OK fixed 0bf73a0

File: src/components/display/fewlines/RadioButton/RadioButton.tsx (hunk 3 above)
Comment (Hamza-A) [reply src/components/display/fewlines/RadioButton/RadioButton.tsx]: OK fixed 0bf73a0

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**